ci-operator/step-registry/hypershift/openstack/e2e/execute/hypershift-openstack-e2e-execute-ref.yaml

```diff
@@ -13,7 +13,7 @@ ref:
       name: release:initial
     - env: CI_HYPERSHIFT_OPERATOR
       name: pipeline:hypershift-operator
-  from: test-bin
+  from: hypershift-tests
```

Comment:
why this change?

Reply:
I have to, the test-bin image is only in Hypershift jobs config. hypershift-tests image is the one that we should use I think.